CONTRIBUTING.md

```diff
@@ -41,240 +37,98 @@ There are several docker compose setups to aid in speeding up the development.
   - The **docker-compose.yml** file contains the most likely setup with development setups for both the api and web projets. This should be used when working on both the api and the web front end. You can run this using **run-api-web.sh**.
   - The **docker-compose.full.yml** contains a setup with all of the containers needed to run Doubtfire with overseer. This requires access to the overseer repository. You can run this using **run-full.sh**
 
-### Working with Docker Compose
-
-To get started:
-
-1. Fork [doubtfire-deploy](https://github.com/doubtfire-lms/doubtfire-deploy), [doubtfire-api](https://github.com/doubtfire-lms/doubtfire-api), and [doubtfire-web](https://github.com/doubtfire-lms/doubtfire-web)
-
-    To push your contributions, you will need a fork of each repository. Contributions can then be made by making pull requests back into the main repositories.
-
-2. Clone your [doubtfire-deploy](https://github.com/doubtfire-lms/doubtfire-deploy). Make sure to fetch submodules to get the subprojects.
+## Getting Started
 
-    `git clone --recurse-submodules https://github.com/YOUR_USERNAME/doubtfire-deploy`
+To run OnTrack locally on your machine:
```

Comment:
**suggestion:** This might be a bit misleading. Reading this makes me think of installing locally on the app on the host machine, not docker. I guess we can drop this line altogether. What do you think?

Reply:
That's a good point. Will address this.

Reply:
Could you confirm if this has been addressed?

Reply:
Could you confirm if this has been addressed?

Reply:
I can confirm - was waiting to hear back about your thoughts on `Getting Started` before pushing a new commit. Thanks!